spec/image_optim_spec.rb

```diff
@@ -89,6 +89,41 @@ def temp_copy(image)
           end
         end
       end
+
+      describe 'skipping larger' do
+        it 'skips larger files' do
+          image_optim = ImageOptim.new(:skip_bigger => true)
+          original = double(:size => 12_345)
+          optimized = double(:size => 100_000)
+
+          allow(ImageOptim::ImagePath).to receive(:convert).and_return original
+
+          expect(image_optim).to receive(:workers_for_image).with(original).
```

Comment:
`allow` instead of `expect` and it returns either an `Array` or `nil` so `and_return([])` would be better